examples/openshift-origin/create.sh

```diff
@@ -110,12 +110,12 @@ echo "sudo cat /srv/kubernetes/server.key; exit;" | gcloud compute ssh ${KUBE_GC
 # The following insertion will fail if indentation changes
 sed -i -e 's/publicKeyFiles:.*$/publicKeyFiles:/g' -e '/publicKeyFiles:/a \ \ - serviceaccounts.private.key' ${OPENSHIFT_CONFIG}/master-config.yaml
 
-docker run -it --privileged -e="KUBECONFIG=/config/admin.kubeconfig" -v ${OPENSHIFT_CONFIG}:/config openshift/origin cli secrets new openshift-config /config -o json &> ${OPENSHIFT_EXAMPLE}/secret.json
-kubectl create -f ${OPENSHIFT_EXAMPLE}/secret.json --namespace='openshift-origin'
+docker run -it --privileged -e KUBECONFIG=/kubeconfig -v ${HOME}/.kube/config:/kubeconfig -v ${OPENSHIFT_CONFIG}:/config openshift/origin:v1.0.3 cli secrets new openshift-config /config -o json &> ${OPENSHIFT_EXAMPLE}/secret.json
```

Comment:
I was a bit worried about the file path issue with the bot. It's in one of the tests.

Changing kubeconfig to just the kubeconfig inside ${OPENSHIFT_EXAMPLE}/config doesn't work alone. Maybe it's just pinning to v1.0.3.

I think it's okay to pin to this because generating secrets shouldn't change much between versions.